docs/class-definitions.md

```diff
@@ -544,3 +546,155 @@ Represent data that can be associated with a cursor update.
 ```ts
 type CursorData = Record<string, unknown>;
 ```
+
+## Component Locking
+
+Provides a mechanism to "lock" a component, reducing the chances of conflict in an application whilst being edited by multiple members. Inherits from [EventEmitter](/docs/usage.md#event-emitters).
+
+### Methods
+
+#### acquire
+
+Send a request to acquire a lock. Returns a Promise which resolves once the request has been sent. A successful response returns a `pending` [Lock](#lock). An error will be thrown if a lock request with a status of `pending` or `locked` already exists.
+
+When a lock is confirmed `locked`, an `update` event will be emitted. Hence to handle lock acquisition, `acquire` needs to always be used together with `subscribe`. 
+
+```ts
+type acquire = (lockId: string) => Promise<Lock>;
+```
+
+Example:
+
+```ts
+const id = "/slide/1/element/3";
+const lockRequest = await space.locks.acquire(id);
+```
+
+#### release
+
+Releases a previously requested lock.
+
+```ts
+type release = (lockId: string) => Promise<void>;
+```
+
+Example:
+
+```ts
+const id = "/slide/1/element/3";
+await space.locks.release(id);
+```
+
+#### subscribe
```

Comment:
```suggestion
#### subscribe()
```

Reply:
I think we should move subscribe above release so it makes sense with the sentence where you say acquire should be used with subscribe. It's also worth linking `subscribe()` to the relevant section.